.github/workflows/build_test_deploy.yml

```
@@ -1463,7 +1463,7 @@ jobs:
           rm -rf test
       - name: Build
         id: build
-        uses: vmactions/freebsd-vm@v0.2.0
+        uses: vmactions/freebsd-vm@v0
```

```suggestion
        uses: vmactions/freebsd-vm@v0.2.3
```

Lets use a specific version

---

I guess the reason the `vmactions/freebsd-vm` prefers the major version is to always able to use the latest changes/fixes, While the version of FreeBSD to be installed will remain unchanged.

---

Let's continue to keep this pinned for now, if we have to keep bumping this a lot we can consider leaving at `v0`.